lib/components/molecules/CookieDisclaimer/CookieDisclaimer.mock.js

```diff
@@ -0,0 +1,40 @@
+import { action } from '@storybook/addon-actions';
+
+const stickyCookieDisclaimer = {
+  showDisclaimer: true,
+  acceptBtn: {
```

Comment:
I think you can set these things inline on the buttons. We know which action is primary and secondary

Reply:
Updated.